src/xcode/ENA/ENA/Resources/Localization/ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,888 @@
+
+"ExposureNotificationSetting_AccLabel_Disabled" = "Один человек деактивировал определение контакта на своем устройстве, то есть его контакты с двумя другими людьми не будут регистрироваться.";
+
+"ExposureNotificationSetting_AccLabel_BluetoothOff" = "Один человек выключил Bluetooth на своем устройстве, то есть его контакты с двумя другими людьми не будут регистрироваться.";
+
+"ExposureNotificationSetting_AccLabel_InternetOff" = "Один человек отключил соединение с Интернетом на своем устройстве, поэтому определить его последние контакты не удалось.";
+
+"ExposureNotificationSetting_Activate_Action" = "Принимаю";
+
+"ExposureNotificationSetting_Dismiss_Action" = "Отмена";
+
+/* Home Navigation Bar */
+"Home_LeftBarButton_description" = "Логотип Corona-Warn-App";
+
+"Home_RightBarButton_description" = "Информация";
+
+/* Home Active card */
+"Home_Activate_Card_On_Title" = "Определение контакта активно";
+
+"Home_Activate_Card_Off_Title" = "Определение контакта остановлено";
+
+"Home_Activate_Card_Bluetooth_Off_Title" = "Bluetooth выключен";
+
+"Home_Activate_Card_Internet_Off_Title" = "Нет соединения с Интернетом";
+
+/* Home common card */
+"Home_Risk_Date_Item_Title" = "Обновлено: %@";
+
+"Home_Risk_No_Date_Title" = "Неизвестно";
+
+"Home_Risk_Period_Update_Title" = "Обновление каждые %@\U00A0ч";
+
+"Home_Risk_Period_Disabled_Button_Title" = "Обновление только каждые %@\U00A0ч";
+
+/* Home Unknown card */
+"Home_Risk_Unknown_Title" = "Неизвестный риск";
+
+"Home_RiskCard_Unknown_Item_Title" = "С момента включения определения контакта не прошло достаточно времени, чтобы рассчитать Ваш риск заражения.";
+
+"Home_RiskCard_Unknown_Button" = "Обновить";
+
+/* Home Inactive card */
+"Home_Risk_Inactive_NoCalculation_Possible_Title" = "Определение контакта остановлено";
+
+"Home_Risk_Inactive_Outdated_Results_Title" = "Определение контакта невозможно";
+
+"Home_Risk_Inactive_NoCalculation_Possible_Body" = "Активируйте определение контакта, чтобы оценить Ваш статус риска сегодня.";
+
+"Home_Risk_Inactive_Outdated_Results_Body" = "Не удается обновить определение контакта более 24 часов.";
+
+"Home_Risk_Inactive_Activate_Item_Title" = "Последнее определение контакта: %@";
+
+"Home_Risk_Inactive_Activate_Item_Unknown_Title" = "Неизвестный риск";
+
+"Home_Risk_Inactive_Activate_Item_Low_Title" = "Низкий риск";
+
+"Home_Risk_Inactive_Activate_Item_High_Title" = "Повышенный риск";
+
+"Home_Risk_Inactive_NoCalculation_Possible_Button" = "Активировать определение контакта";
+
+"Home_Risk_Inactive_Outdated_Results_Button" = "Активировать определение контакта";
+
+/* Home Low card */
+"Home_Risk_Low_Title" = "Низкий риск";
+
+"Home_Risk_Low_SaveDays_Item_Title" = "Дней активности: %@ из %@";
+
+"Home_Risk_Low_Button" = "Обновить";
+
+/* Home High card */
+"Home_Risk_High_Title" = "Повышенный риск";
+
+"Home_Risk_High_Button" = "Обновить";
+
+/* Home Status Check */
+"Home_Risk_Status_Check_Title" = "Проверка выполняется...";
```

Comment:
```suggestion
"Home_Risk_Status_Check_Title" = "Проверка выполняется…";
```
… instead of 2 dots?

Reply:
our translation system can't handle unicode symbols unfortunately